modules/api/docs/LorisRESTAPI_v0.0.4-dev.md

```diff
@@ -823,90 +823,192 @@ The JSON object is of the form:
+                  "SliceThickness" : "1",
+                  "Modality" : "MR",
+                  "SeriesUID" : "1.7.8.9.1296",
+                }
+            ]
         }
-    ]    
+    ]
 }
 ```
 
-The `Modality` header in the SeriesInfo is either `MR` or `PT` for MRI or PET 
+The `Modality` header in the SeriesInfo is either `MR` or `PT` for MRI or PET
 scans, respectively.
 
+```
+POST /candidates/$CandID/$VisitLabel/dicoms
+```
+
+The body of the POST request should be encoded as a form data and contain a
+field `File` with the imaging file, and `Json`, with the following keys and
+values:
+
+```js
+{
+    "CandID": $CandID,
+    "PSCID": $PSCID,
+    "VisitLabel": $VisitLabel,
+    "IsPhantom": boolean
+}
+```
+
+An optionnal header 'LORIS-Overwrite' with the value 'overwrite' can be used to
+overwrite an existing file.
+A successful request will be answered by a `303 See Other` response with its
+`Location` header pointing to the processes list of the new upload.
+(See 5.3 Tar Level processes)
+
 ### 5.2 Tar Level Data
+
+This section describe how to upload DICOM studies and how to start and monitor
+`mri_upload` processes.
+DICOM studies that have been successfully uploaded and processed can be downloaded with
+the following `GET` request:
+
 ```
-GET /candidates/$CandID/$VisitLabel/dicoms/$Tarname
+GET /candidates/$CandID/$VisitLabel/dicoms/$ArchiveName
 ```
 
-Returns/Downloads a `tar` file which contains a `.meta` and a `.log` text 
+Returns/Downloads a `tar` file which contains a `.meta` and a `.log` text
 files, and a `.tar.gz` of the raw DICOM data as acquired during the candidate
-scanning session, and as retrieved from `/candidates/$CandID/$Visit/dicoms`.
+scanning session, and as retrieved from `/candidates/$CandID/$VisitLabel/dicoms`.
 
-Only `GET` is currently supported.
+To get a list of the processes and their status for a given DICOM study previously uploaded use the following:
+
+```
+GET /candidates/$CandID/$VisitLabel/dicoms/$TarName/processes
+```
+
+The response contains all `mri_upload` attempts with the specified `$tarname`. And for
+each of them, a list of processes status.
+Response shape:
+
+```js
+{
+  "MriUploads": [
+    {
+      "MriUploadID": 123,
+      "Processes": [
+        {
+          "END_TIME": "YYYY-MM-DD hh:mm:ss",
+          "EXIT_CODE": "0",
+          "ID": "1",
+          "PID": "24971",
+          "PROGRESS": "text"
+          "STATE": "SUCCESS|RUNNING|ERROR"
+        },
+        ...
+      ]
+    },
+    ...
+  ]
+}
+```
+
+** An empty `Processes` array means that there has never been a process launched on
+that MRI upload.
+To start an MRI upload process on a previously uploaded DICOM study, a POST request
+containing the `MriUploadID` attribute in the request body should be sent.
+
+```
+POST /candidates/$CandID/$VisitLabel/dicoms/$TarName/processes
+```
+
+The request body must contain:
+
+```js
+{
+  "ProcessType": "mri_upload",
+  "MriUploadID": 123
+}
+```
```

Comment:
Here I get:
```
[Tue Oct 01 15:54:01.123601 2024] [php:notice] [pid 12396] [client ::1:51779] Cannot run command source '/Users/cmadjar/Development/Github/Loris-MRI/environment_loris' ; '/Users/cmadjar/Development/Github/Loris-MRI/uploadNeuroDB/imaging_upload_file.pl' -upload_id '127' -profile 'prod' '/Users/cmadjar/Data/LORIS_imaging_data/incoming/MTL001_300001_V2.tgz'
```

The paths are correct (even if they look unusual). Is there a config I need to set or something I may be missing?